src/openforms/forms/models/form.py

```diff
@@ -182,3 +184,72 @@ def get_keys_for_email_confirmation(self) -> List[str]:
     def iter_components(self, recursive=True):
         for form_step in self.formstep_set.select_related("form_definition"):
             yield from form_step.iter_components(recursive=recursive)
+
+    def all_versions(self):
+        from .form_version import FormVersion
+
+        return FormVersion.objects.filter(form=self)
```

Comment:
This seems to be unused, and the better approach would be to use `form.formversion_set.all()` instead anyway.